static/admin/index.html

```diff
@@ -26,7 +26,7 @@
 </div>
 <script id="admin-config-script" type="text/javascript">
   var env = {
-      siteId: "0pvklfck1gkdpg",
+      siteId: "g0tpjekrmqgtkw",
```

Comment:
Note: Might not want to merge the updated Forestry site ID, since changes will be available in the "Saga" site on Forestry after merging the PR.